### ISSUE TITLE
Aquatic Mobs

### ISSUE THREAD

**Issue by ktclarke** [SlimeDog/AggressiveAnimals#29]
Let's be honest, nothing is more amusing than seeing a player killed by angry cod. Hilarious.

I would love an added rule in the config to limit aquatic targeting to only players in the water. This would both bring a false sense of calm to the water (until you fall in...) and would prevent the obvious farming exploit of safely standing at the shoreline and attacking the aggressive fish.

**Comment by SlimeDog** [comment]
Thanks more the suggestion. We'll have a look at it. It's not a high priority, since the acquisition distance can be set to effectively limit attacks to being in (or very near) the water.

**Comment by SlimeDog** [comment]
Part A: This should be the default.

Part B: Make it an option. This would require additions to `config.yml` and code to support them.
```
defaults:
...
  attack-only-in-water: true       # If true, aquatic mobs attack only if the targeted player is in the water with them
...
mobs:
...
  axolotl:
    attack-only-in-water: true
...
  frog:
    attack-only-in-water: true
...
  glow_squid:
    attack-only-in-water: true
...
  pufferfish:
    attack-only-in-water: true
...
  salmon:
    attack-only-in-water: true
...
  squid:
    attack-only-in-water: true
...
  tadpole:
    attack-only-in-water: true
...
  tropical_fish:
    attack-only-in-water: true
...
  turtle:
    attack-only-in-water: true
...
```

**Comment by SlimeDog** [comment]
Included in AA 1.5.0
